src/app/landing-page/landing-section/landing-section.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-landing-section',
+  templateUrl: './landing-section.component.html',
+  styleUrls: ['./landing-section.component.scss']
+})
+export class LandingSectionComponent implements OnInit {
+
+  constructor() { }
+
+  ngOnInit(): void {
+  }
```

Comment:
If we are doing consistent style, then maybe we should write `ngOnInit(): void { }` in the same line as well?

Reply:
Actually constructor and ngOnInit aren't used right now so they can be deleted

Reply:
Done 234cffea2932bf201c0dcc986734ae61f2398023